core/src/banking_stage/transaction_scheduler/greedy_scheduler.rs

```diff
@@ -124,6 +124,12 @@ impl<Tx: TransactionWithMeta> Scheduler<Tx> for GreedyScheduler<Tx> {
         let mut num_unschedulable_conflicts: usize = 0;
         let mut num_unschedulable_threads: usize = 0;
 
+        // Main scheduling loop: greedily schedule transactions in priority order.
+        // The loop continues while:
+        // - We have budget remaining (compute units)
+        // - We haven't exceeded the scan limit (prevents infinite loops)
+        // - There are schedulable threads available
+        // - There are transactions in the container
```

Comment:
not sure I see any value in the comment here - this is just a text description of the code and easily falls out of date.